content/includes/survey-tools.html

```diff
@@ -0,0 +1,8 @@
+{% if buildtype === 'staging' %}
+  {% assign loa3_entrynames = 'burials, claims-status, letters, post-911-gib-status, hca, health-records, messaging, rx, pensions, profile-360, claims-status' %}
```

Comment:
Can't assign arrays in liquid, but according to the docs, we can split a string into an assignment:

```
{% assign loa3_entrynames = "burials, claims-status, ..." | split: ", " %}
   {% if entryname in loa3_entrynames %}
    ...
```

so we don't end up with any potential issues with an loa1 entryname with a name that's a subset of an loa3 name. 

Minor note - claims-status is mentioned twice.

Reply:
Ah I must have missed the `in` operator! Updating now.